CHANGELOG.md

```diff
@@ -15,6 +16,7 @@
 * Add `schedulerName` to `podTemplate` (#2114)
 * Allow overriding the auto-detected Kubernetes version
 * Garbage Collection (GC) logging disabled by default
+* Expose JMX port on Kafka brokers via an internal service
```

Comment:
As we released the 0.15.0 in the mean time, could you move this to 0.16.0 section?

Reply:
Will do so!